tensorflow/contrib/distributions/python/kernel_tests/quantized_distribution_test.py

```diff
@@ -361,15 +362,14 @@ def testProbAndGradGivesFiniteResultsForCommonEvents(self):
 
   def testLowerCutoffMustBeBelowUpperCutoffOrWeRaise(self):
     with self.cached_session():
-      qdist = distributions.QuantizedDistribution(
-          distribution=distributions.Normal(loc=0., scale=1.),
-          low=1.,  # not strictly less than high.
-          high=1.,
-          validate_args=True)
-
-      self.assertTrue(qdist.validate_args)  # Default is True.
-      with self.assertRaisesOpError("must be strictly less"):
-        qdist.sample().eval()
+      with self.assertRaisesWithPredicateMatch(errors.InvalidArgumentError,
+                                               "must be strictly less"):
+        qdist = distributions.QuantizedDistribution(
+            distribution=distributions.Normal(loc=0., scale=1.),
```

Comment:
Unused variable 'qdist' [unused-variable]

Reply:
Changed `qdist` to `_`